db/seeds.rb

```diff
@@ -0,0 +1,43 @@
+# my user info, etc.
```

Comment:
You need to be using `Model.create!`  with a `!` because `Model.create` does not raise an error when the create fails. So either check the return value or use `Model.create!`